flexget/plugins/notifiers/notify.py

```diff
@@ -156,6 +162,53 @@ def on_task_start(self, task, config):
     on_task_exit = send_notification
 
 
+class NotifyEntries(NotifyBase):
+    def on_task_exit(self, task, config):
+        config['scope'] = 'entries'
+        plugin.get_plugin_by_name('notify').instance.send_notification(task, config)
+
+
+class NotifyTask(NotifyBase):
+    def on_task_exit(self, task, config):
+        config['scope'] = 'task'
+        plugin.get_plugin_by_name('notify').instance.send_notification(task, config)
+
+
+class NotifyAbort(NotifyBase):
+    def on_task_abort(self, task, config):
+        if task.silent_abort:
+            return
+
+        title = 'Task {{ task_name }} has aborted!'
+        message = 'Reason: {{ task.abort_reason }}'
+        notify_config = {'to': config['to'],
+                         'scope': 'task',
+                         'title': title,
+                         'message': message}
+        log.debug('sending abort notification')
+        plugin.get_plugin_by_name('notify').instance.send_notification(task, notify_config)
+
+
+class NotifyCrash(NotifyBase):
+    def on_task_abort(self, task, config):
```

Comment:
I was thinking notify_crash would be a root level plugin, such that it would notify you about crashes in the scheduler or other contexts outside of tasks as well. Right now this is just the same as notify_abort, no?